src/modules/render.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable*/
+import { todosArr, listBox } from "../index.js";
+import clearCompletedTodo from "./clear.js";
+
+export default function displayTodo() {
+  let htmlCode = "";
+  todosArr.forEach((todo, index) => {
+    htmlCode += `<div class="list-items" id= '${index}'>
+    
+    <i class="bi ${
+      todo.completed ? "bi-check-circle-fill" : "bi-circle"
+    }" data-action = "check" id="flex-start"></i> 
```

Comment:
Very interesting line of code here. But can it be space-less equals signs, please? 'data-action="check"' instead of data-action = "check".